doc/users/project/mission.rst

```diff
@@ -0,0 +1,28 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  We judge our success based on how effectively we enable our users, across
+scientific, industrial, commercial, and educational domains, to achieve their
+goals.
+
+We believe that a diverse developer community creates the best software, and we
+welcome anyone who shares our mission and values, as described in the code of
+conduct.
+
+
+Adapting the requirements :ref:`laid out by John Hunter
+<project_history>` Matplotlib should:
+
+* Support all users of the Scientific Python Ecosystem;
```

Comment:
```suggestion
* Support all users of the Scientific Python ecosystem;
```
Link to https://scientific-python.org/ ? (Which I cannot really remember how to do.)